Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "PythonCall"
 uuid = "6099a3de-0909-46bc-b1f4-468b9a2dfc0d"
 authors = ["Christopher Doris <github.com/cjdoris>"]
-version = "0.9.6"
+version = "0.9.7"
```

Comment:
Please revert this, version bumping is more complicated that this, due to the Python component.